package.json

```diff
@@ -34,10 +34,13 @@
     "dedent": "^0.7.0",
     "dotparser": "^0.4.0",
     "enumify": "^2.0.0",
+    "graphiql": "^0.17.5",
     "graphql": "^14.5.8",
     "graphql-tag": "^2.10.1",
     "jshint": "^2.10.2",
     "nprogress": "^0.2.0",
+    "react": "^15.6.2",
+    "react-dom": "^15.6.2",
```

Comment:
Looks weird to have this in a Vue app, but we wouldn't be the first ones to try

- https://github.com/vuejs/Discussion/issues/1301

And actually works. The generated app JS size will increase, but for the user it should still be loaded dynamically when the GraphiQL view is accessed.

The same already happened anyway, and via a CDN, instead of local files. So now GraphiQL should work even if accessed in an environment without external connectivity (related to #386).